zbm-builder.sh

```diff
@@ -73,15 +148,17 @@ while getopts "hHLCdt:B:" opt; do
   esac
 done
 
-if ! command -v "${PODMAN}" >/dev/null 2>&1; then
-  echo "ERROR: container front-end ${PODMAN} not found"
-  exit 1
+if [ -n "${BUILD_DIRECTORY}" ]; then
+  VOLUME_ARGS+=( "-v" "${BUILD_DIRECTORY}:/build" )
+else  
+  BUILD_DIRECTORY="${PWD}"
+  VOLUME_ARGS+=( "-v" "${PWD}:/build" )
 fi
```

Comment:
Now any configured mounts must be handled.
```suggestion
if ! BUILD_DIRECTORY="$( sanitise_path "${BUILD_DIRECTORY}" )"; then
  echo "ERROR: build directory does not exist"
  exit 1
fi
VOLUME_ARGS+=( "-v" "${BUILD_DIRECTORY}:/build" )

if [ -n "${LOCAL_REPO}" ]; then
  if ! LOCAL_REPO="$( sanitise_path "${LOCAL_REPO}" )"; then
    echo "ERROR: local repository does not exist"
    exit 1
  fi
  VOLUME_ARGS+=( "-v" "${LOCAL_REPO}:/zbm" )
fi
```

Reply:
:+1: